specification/src/main/asciidoc/platform/PlatformOverview.adoc

```diff
@@ -199,8 +199,10 @@ bean container.
 ==== Container Requirements
 
 This specification requires that containers
-provide a Java Compatible™ runtime environment, as defined by the Java
-Platform, Standard Edition, v8 specification (Java SE). The applet
+support execution in a Java™ runtime environment, as defined by the Java
```

Comment:
@kwsutter On #287 we talked about describing testing requirements instead of the 'provide' language. We could do that but I wasn't sure whether this document is meant to cover compliance certification requirements for implementors, as opposed to what users of the specification would expect.

Reply:
@bstansberry I don't think we want the Platform Spec to get into the compliance certification game (or rules).  The Spec should outline the language requirements.  How that is verified or tested should be left up to the TCK documentation.